assets/js/translator.js

```diff
@@ -125,7 +127,7 @@ if(modeEnabled('translation')) {
 
             timer = setTimeout(function () {
                 if($('#instantTranslation').prop('checked')) {
-                    translateText();
+                    translate();
```

Comment:
Hm, I don't want instant translation to apply to documents. Would this make that happen?